june/19/karankartikeya.md

```diff
@@ -0,0 +1,9 @@
+# Yesterday
+- updated my team listing page preview
+- tried learning the backend from the reference code repo
+
+# Today
+- Will learn more about the npm packages and the .json packages use
```

Comment:
try running `npm init` in empty directory

Reply:
Yes,thank you
I am taking the help of the video uploaded by you on the youtube to understand these commands.